src/Project/ProjectVersionGit.php

```diff
@@ -126,6 +134,7 @@ protected function queryGitData()
         }
         $data['url'] = strtr($url, array('.git' => '', 'ssh://' => $R, 'git://' => $R, 'ftp://' => $R, 'ftps://' => $R, 'rsync://' => $R));
         $data['tag'] = exec('git describe --tags --always');
+        $data['closestTag'] = exec('git describe --tags --always --abbrev=0');
```

Comment:
In case there is no tag, this will return the hash (because of `--always`). Would we prefer an error instead (no `--always`)?